.vscode/launch.json

```diff
@@ -0,0 +1,61 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Short Query (Spotify)",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/cli.js",
+            "args": [
+                "spotify:artist:6M2wZ9GZgrQXHCFfjv46we",
+                "--no-auth"
+            ],
```

Comment:
Let's substitute this for `npm test spotify.track` or `npm test spotify.artist`

Reply:
Sounds good. Replaced the launch configurations with the `npm test ...` calls.

The console isn't showing the full output for this, which makes debugging a bit trickier. I think there is a launch setting in vscode that will attach it to stdout so I'm gonna try to find that.